### PR TITLE
[Docker] Correct keyserver and key id in the Dockerfile

### LINKED ISSUE #1586: sks-keyservers.net is deprecated
ITNOA

**Describe the bug**
sks-keyservers.net does not exist in this year, so `gpg --keyserver ha.pool.sks-keyservers.net` does not work

**To Reproduce**
```
cd docker/master/
docker build .

```
**Expected behavior**
`docker build .` create docker image correctly

### PR DESCRIPTION
**Pull request by soroshsabz**
ITNOA

This issue resolved #1586 and merge from #1364 

**My work**
* I updated your branch to head of master.
* I correct keyserver
* I persist gpg key in layer

### CODE REVIEW

**Comment by soroshsabz** [comment]
related to #1365

File: docker/master/Dockerfile [hunk 1]
```diff
@@ -8,7 +9,9 @@ FROM debian:jessie
 RUN groupadd -r freeswitch --gid=999 && useradd -r -g freeswitch --uid=999 freeswitch
 
 # grab gosu for easy step-down from root
-RUN gpg --keyserver ha.pool.sks-keyservers.net --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4
+RUN apt-get update && apt-get install -y --no-install-recommends dirmngr gnupg2 \
+    && gpg2 --keyserver hkp://keyserver.ubuntu.com --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4 \
```
Comment (andywolk) [inline docker/master/Dockerfile:13]: Keep in mind that the key was changed. Also SignalWire token is required now.
https://freeswitch.org/confluence/display/FREESWITCH/Debian
Here is where you can find the key without auth.
https://keys.openpgp.org/search?q=655DA1341B5207915210AFE936B4249FA7B0FB03

**Review by andywolk — CHANGES_REQUESTED** [review]

File: docker/master/Dockerfile (hunk 1 above)
Comment (andywolk) [reply docker/master/Dockerfile:13]: You can try this
```
gpg --keyserver hkp://keyserver.ubuntu.com --recv-keys 655DA1341B5207915210AFE936B4249FA7B0FB03
```

Comment (soroshsabz) [reply docker/master/Dockerfile:13]: Thanks for reviewing this PR, Is there any alternative way for getting package without needing PAT? I think public open source application must does not need to dependent to specific user.

Comment (soroshsabz) [reply docker/master/Dockerfile:13]: @andywolk I get PAT from ARG in Dockerfile to does not mandatory expose private info in GitHub

Comment (andywolk) [reply docker/master/Dockerfile:13]: > Did I have to upload my PAT to this PR? @andywolk

Do not upload your PAT. 
Yes you can use `ARG` in the Dockerfile.

**Comment by soroshsabz** [comment]
I think, I can resolve it, Please review it again

thanks

File: docker/master/Dockerfile [hunk 2]
```diff
@@ -8,7 +10,11 @@ FROM debian:jessie
...
```
Comment (andywolk) [inline docker/master/Dockerfile:14]: What's the point of this key?

Comment (soroshsabz) [reply docker/master/Dockerfile:14]: It is exist from past

Comment (soroshsabz) [reply docker/master/Dockerfile:14]: @andywolk I think it is used for `gpg --verify /usr/local/bin/gosu.asc`

Comment (soroshsabz) [reply docker/master/Dockerfile:14]: @andywolk I think it is used for `gpg --verify /usr/local/bin/gosu.asc`

Comment (andywolk) [reply docker/master/Dockerfile:14]: > @andywolk I think it is used for `gpg --verify /usr/local/bin/gosu.asc`

no

**Review by andywolk — APPROVED** [review]

File: docker/master/Dockerfile [hunk 3]
```diff
@@ -8,7 +10,11 @@ FROM debian:jessie
...
+    && gpg2 --keyserver hkp://keyserver.ubuntu.com --recv-keys 655DA1341B5207915210AFE936B4249FA7B0FB03 \
+    && gpg2 --output /usr/share/keyrings/signalwire-freeswitch-repo.gpg --export 655DA1341B5207915210AFE936B4249FA7B0FB03 \
+    && apt-get purge -y --auto-remove gnupg2
```
Comment (andywolk) [inline docker/master/Dockerfile:17]: Does this break `gpg --verify /usr/local/bin/gosu.asc` ?

Comment (soroshsabz) [reply docker/master/Dockerfile:17]: if `gpg2 --keyserver hkp://keyserver.ubuntu.com --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4` remove, I think it is break, but now I do not see any break

Am I correct?

**Comment by soroshsabz** [comment]
@andywolk  Thanks for merging :)

File: docker/master/Dockerfile (hunk 3 above)
Comment (andywolk) [reply docker/master/Dockerfile:17]: Do you remove gnupg2 before calling gpg verify?

Comment (soroshsabz) [reply docker/master/Dockerfile:17]: I try to remove extra layer, with removing `RUN` please wait

Comment (soroshsabz) [reply docker/master/Dockerfile:17]: @andywolk  I submit new PR #1589

**Pull request merged**